### ISSUE TITLE
Output streams

### ISSUE THREAD

**Issue by g105b** [phpgt/Cli#5]
When a command outputs something, it needs to be grouped into its own output stream.

Each stream can be provided a different colour, after #4 is implemented. For non-coloured terminals, the name of the command that is being output can be shown along with a timestamp before the message.

For example, two long-running processes "time" and "status" that make output every few seconds:

```
[time] 2020-04-05 17:25:53
The time is twenty five minutes past five.

[status] 2020-04-05 17:25:59
OK

[time] 2020-04-05 17:26:03
The time is twenty six minutes past five.

[status]  2020-04-05 17:26:10
OK
```

**Comment by g105b** [comment]
This is too confusing since Cli and Daemon were split - the functionality seems like how Daemon already works. If it's needed in the future I will create a new issue on the appropriate repo.
